lib/get-changesets.js

```diff
@@ -180,9 +182,8 @@ const getChangesetMetadata = async (changesetId) => {
 };
 
 const checkLatest = async (max) => {
-  const { PrimaryAdiffBaseUrl, SecondaryAdiffBaseUrl } = process.env;
-  const primaryUrl = `${PrimaryAdiffBaseUrl}latest`;
-  const secondaryUrl = `${SecondaryAdiffBaseUrl}latest`;
+  const primaryUrl = `${OverpassPrimaryUrl}/aug-diffs/newest`;
+  const secondaryUrl = `${OverpassSecondaryUrl}/api/augmented_diff_status`;
```

Comment:
What's the reason these two URLs are different? Is this different versions of Overpass or so? Should we allow the state path to be configurable via environment variables as well?

Reply:
Oh! I was used to use the /newest, but now I see the /api/augmented_diff_status also works on our overpass. I'm going to replace it.